### DISCUSSION TITLE
VDH still detects media from Twitch even after being blacklisted

### DISCUSSION THREAD

**Discussion by XangelMusic** [aclap-dev/video-downloadhelper#367] [Help and Q&A]
### Your question

As the title states, I still get media results when going to Twitch even though I added "twitch.tv" to the blacklist.

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 75052b43f0614124
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Josue Cerna",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Mattmjs** [comment]
I have no detection of live streams after adding _twitch.tv_ & _www.twitch.tv_ to the blacklist. You're referring to live streams correct ?
There is also non live stream media on twitch.

**Reply by XangelMusic** [reply]
Okay so on both a live stream and the front page, I get results in the VDH. The icon isn't even greyed out.

What did you mean by non-live stream media? Is that what VDH is getting?

**Reply by Mattmjs** [reply]
While I blacklisted the live streams, there are also short clips which were detected that use other domains.
What do you see in the blacklist.

**Reply by XangelMusic** [reply]
I only blacklisted twitch.tv as the domain. Just added www.twitch.tv and it works now.

In order words, twitch.tv wasn't sufficient enough.

**Reply by DBLClick** [reply]
Showing why VDH needs the blacklisting to allow for the entire domains.   Streams are still being detected on Twitch CDN because they use generated host names on the CDN.  WIthout blocking the entire domain.  We will always have detection.
